src/main/webapp/i18n/en/exercise-actions.json

```diff
@@ -46,6 +46,8 @@
             "cloneRatedRepository": "Clone your rated repository for this exercise:",
             "clonePracticeRepository": "Clone your practice repository for this exercise:",
             "cloneExerciseRepository": "Clone a repository for this exercise",
+            "waitForData": "Data is still being loaded. Please wait.",
```

Comment:
This text is also a bit vague. I think you can be more concrete and mention the setup of the access to the repository is still being initialized

Reply:
```suggestion
            "waitForData": "Setting up access to the repository. Please wait.",
```